tuf/client/updater.py

```diff
@@ -733,6 +733,12 @@ def __init__(self, repository_name, repository_mirrors):
     # Set the path for the current set of metadata files.
     repositories_directory = tuf.settings.repositories_directory
     repository_directory = os.path.join(repositories_directory, self.repository_name)
+
+    # raise MissingLocalRepository if the repo does not exist at all.
+    if not os.path.exists(repository_directory):
+      raise tuf.exceptions.MissingLocalRepositoryError('Local repository ' +
+        repr(repository_directory) + 'does not exist.')
```

Comment:
NIT: Maybe add another space before `does not exists` to make it more readable, like that:
`repr(repository_directory) + ' does not exist.')`